gradle.properties

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+#
+# WSO2 LLC. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+group=org.ballerinalang
+version=0.1.0-SNAPSHOT
+ballerinaLangVersion=2201.8.0-20230726-145300-b2bdf796
+org.gradle.caching=true
+org.gradle.parallel=true
+org.gradle.jvmargs='-Dfile.encoding=UTF-8'
+org.gradle.workers.max=3
+
+# Native Dependency Versions
+zipkinVersion=2.24.0
+zipkinReporterVersion=2.16.0
+okioVersion=3.5.0
+okhttpVersion=4.11.0
+kotlinVersion=1.2.71
+kotlinStdlibVersion=1.8.22
+openTelemetryVersion=1.0.0
+openTelemetryAlphaVersion=1.0.0-alpha
+guavaVersion=30.1-jre
```

Comment:
Can we use later versions like 32.2-jre? 
Check for other dependencies as well

Reply:
updated the guava version. Other dependencies have the latest stable versions except the opentelemetry version. It has a a lot of work needs to be done to upgrade (same happened with jaeger).